src/applications/lgy/coe/containers/IntroductionPage.jsx

```diff
@@ -5,20 +5,22 @@ import { isLoggedIn } from 'platform/user/selectors';
 import { notLoggedInContent } from './introductionContent/notLoggedInContent.jsx';
 import { loggedInContent } from './introductionContent/loggedInContent.jsx';
 import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+import { CALLSTATUS } from '../constants';
 
 function IntroductionPage(props) {
   let content;
 
   useEffect(() => {
     focusElement('.va-nav-breadcrumbs-list');
   });
+  // Set the content to be the loading indicator
+  content = <LoadingIndicator message="Loading your application..." />;
 
-  if (props.status === 'idle') {
-    content = <LoadingIndicator message="Loading your application..." />;
-  } else if (
-    props.status === 'success' ||
-    props.status === 'failed' ||
-    props.status === 'skip'
+  // Once the coe call is done, render the rest of the content
+  if (
+    props.status === CALLSTATUS.success ||
```

Comment:
I was thinking about this nested if statement last night, and it could be cleaner to create a simple helper, or check a static array: 

```
const coeCallEnded = [CALLSTATUS.failed, CALLSTATUS.success, CALLSTATUS.skip];

if (!props.loggedIn && coeCallEnded.includes(props.status) {
    content = notLoggedInContent(props);
}
if (props.loggedIn && coeCallEnded.includes(props.status) {
    content = loggedInContent();
}
...
```

Probably more opinion than anything else though.

Reply:
This is a really good refactor, incorporating this